syntax/show_commit_diffstat.sublime-syntax

```diff
@@ -6,7 +6,7 @@ hidden: true
 scope: git-savvy.commit-diffstat
 contexts:
   main:
-    - match: ^ (.+) \| +(\d+) (\+*)(-*)$\n?
+    - match: ^ (\S.+) \| +(\d+) (\+*)(-*)$\n?
```

Comment:
`(.+)` will also match space, just to make sure it won't accidentally match sth else.